### ISSUE TITLE
Rails Advanced Topics: Some code blocks use the incorrect language for syntax highlighting

### ISSUE THREAD

**Issue by MaoShizhong** [TheOdinProject/curriculum#28501]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

Some of the code blocks in the Advanced Topics lesson use the incorrect language, leading to incorrect syntax highlighting.

## Acceptance criteria

- [ ] The following code blocks have their languages changed to `erb`
  - Line 209
  - Line 220
  - Line 234

Please comment below if you'd like to be assigned to this issue. Please do not open a PR unless you have been assigned by a maintainer.

### Path

Ruby / Rails

### Lesson Url

https://www.theodinproject.com/lessons/ruby-on-rails-advanced-topics

**Comment by TheSeagullDev** [comment]
I'd be happy to work on this.

**Comment by MaoShizhong** [comment]
All yours
